packages/components/src/popover/index.js

```diff
@@ -328,7 +329,7 @@ const Popover = (
 		}
 
 		return autoUpdate( usedRef, refs.floating.current, update );
-	}, [ anchorRef, anchorRect, getAnchorRect ] );
+	}, [ anchorRef, anchorRect, getAnchorRect, middlewares ] );
```

Comment:
Interesting that this is also not enough to trigger an update of the popover behavior when changing its behavior through enabling/disabling middlewares